### ISSUE TITLE
add draftify result

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia-gadgets/xfdcloser#68]
Requested by Vanamonde93 at https://en.wikipedia.org/wiki/Wikipedia_talk:XFDcloser#RFC%3A_Priorities_for_XFDcloser_development_in_2022

**Comment by NovemLinguae** [comment]
Requested by Daniel at https://en.wikipedia.org/w/index.php?title=Wikipedia_talk:XFDcloser&curid=52907095&diff=1242737312&oldid=1242289802

**Comment by NovemLinguae** [comment]
This could potentially be split into 3 tickets:

* [ ] AFD draftify
* [ ] AFD userfy
* [ ] TFD userfy

**Comment by Sky2042** [comment]
>  TFD userfy

Indeed, draftify is basically never used at TFD. We do regularly userfy templates and sandboxify modules by request.

Templates end up at `User:<requester>/<whatever_name_we_please>`. I tend toward the template name without namespace personally for the `<whatever>`, but you could also add other slashes in there e.g. `/sandbox/`.

Modules end up at `Module:Sandbox/<requester>/<whatever>`, and in this regard I'd use the module name without namespace (since `Module:` is required for all modules).

Transclusions are removed, updated, or commented out/tl'd and each depends on the specific template and which namespaces the transclusions are used in. So the start to that feature is probably just the page moves.

No comment on preferred task organization. I'd guess what I've described is enough to put it in a different task.

**Comment by IlyasLebleu** [comment]
Currently working on it!

**Comment by Trialpearswiki** [comment]
@IlyasLebleu do you still wish to work on this? Otherwise I think I might tackle it.

**Comment by IlyasLebleu** [comment]
I have it mostly ready, just a lot of procrastination for the final debugging!

**Comment by NovemLinguae** [comment]
Requested by OwenX at https://github.com/NovemLinguae/UserScripts/issues/309

**Comment by OwenX-wiki** [comment]
Thanks! I should have checked first. Not surprised it's a frequently requested feature.

**Comment by FaviFake** [comment]
On enwiki, [OwenX had the idea](https://en.wikipedia.org/wiki/User_talk:Novem_Linguae#c-OwenX-20260701152900-FaviFake-20260701144700) of relying on the existing script used to do this. Is this a viable or technically possible option, or is it better to develop the feature from scratch?

> We already have [User:MPGuy2824/MoveToDraft.js](https://en.wikipedia.org/wiki/User:MPGuy2824/MoveToDraft.js), so perhaps invoking that script from XFDcloser with the appropriate parameters is all that's needed here to save the tedious manual step

**Comment by NovemLinguae** [comment]
Probably easier to code this from scratch. But up to whoever writes the patch if they want to copy paste MoveToDraft code in or not.
